rules-reviewed/eap7/eap6/tests/data/data-hsearch/HsearchUtil.java

```diff
@@ -127,7 +131,12 @@ public void main(String[] args) {
         BuildContext context = (BuildContext) searchIntegrator;
         context.getIndexingStrategy();
         
-        System.out.println(IndexWriterSetting.TERM_INDEX_INTERVAL + " test");
```

Comment:
btw @Ladicek this doesn't work at all, because of type argument in {{System.out.println}} method is {{java.lang.String}} type and that doesn't trigger the analysis for types in case of Enum or Enum constant  while Windup parses with AST.